File: wiz-event-mall-java/src/main/java/com/example/wizeventmalljava/domain/member/repository/MemberRepository.java
package com.example.wizeventmalljava.domain.member.repository;

import java.util.Optional;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

import com.example.wizeventmalljava.domain.member.entity.Member;

@Repository
public interface MemberRepository extends JpaRepository<Member, Long> {

	boolean existsByEmail(String email);
	Optional<Member> findByEmail(String email);
}

File: wiz-todo/src/main/kotlin/com/example/wiztodo/domain/member/repository/MemberRepository.kt
package com.example.wiztodo.domain.member.repository

import com.example.wiztodo.domain.member.entity.Member
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface MemberRepository : JpaRepository<Member, Long> {
    fun existsByEmail(email: String): Boolean
    fun findByEmail(email: String): Member?
}